packages/shared/providers.yaml

```diff
@@ -3296,6 +3296,8 @@ intercom:
     token_url: https://api.intercom.io/auth/eagle/token
     proxy:
         base_url: https://api.intercom.io
+        retry:
```

Comment:
Should be all lower case now as per a recent change https://github.com/NangoHQ/nango/pull/3110

Reply:
it's enforced by the CI anyway 👌🏻

Reply:
Aah yes, updated with new [commit](https://github.com/NangoHQ/nango/pull/3122/commits/bd4c26e2ef6027eddfc8fe012e8189d2cd471c76)